tests/wpt/web-platform-tests/tools/wptrunner/wptrunner/executors/executorservodriver.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, session):
             self.session = session
 
         @webdriver.client.command
-        def get_prefs(self, *prefs):
+        def prefs(self, *prefs):
```

Comment:
This change looks like an incorrect global search and replace for get_prefs.

Reply:
@jdm Good catch. I found another one in `webdriver_server/lib.rs` too.